src/Analysis.cpp

```diff
@@ -668,25 +668,27 @@ void CodeAnalyzer::doAnalysis(Function &F, std::string optLoadFileName) {
         }
     }
 
-    CFG cfg;
-    buildCFG(cfg, realBranchOrder, branchInstructionMap);
+    CFG *cfg = new CFG();
+    buildCFG(*cfg, realBranchOrder, branchInstructionMap);
 
     calledMethods.setFunctions(SafeFunctions, ReallocFunctions, MemoryFunctions,
                                annotationHandler);
-    calledMethods.setCFG(&cfg);
+    calledMethods.setCFG(cfg);
     calledMethods.setProgramFunction(programFunction);
     calledMethods.setFunctionInfosManager(functionInfosManager);
     calledMethods.setOptLoadFileName(optLoadFileName);
 
     mustCall.setFunctions(SafeFunctions, ReallocFunctions, MemoryFunctions,
                           annotationHandler);
-    mustCall.setCFG(&cfg);
+    mustCall.setCFG(cfg);
     mustCall.setProgramFunction(programFunction);
     mustCall.setFunctionInfosManager(functionInfosManager);
     mustCall.setOptLoadFileName(optLoadFileName);
 
-    ProgramFunction PostCalledMethods = calledMethods.generatePassResults();
-    ProgramFunction PostMustCalls = mustCall.generatePassResults();
+    ProgramFunction *PostCalledMethods = calledMethods.generatePassResults();
+    ProgramFunction *PostMustCalls = mustCall.generatePassResults();
+
+    /*
```

Comment:
As a rule, I prefer to delete code rather than comment it out. The version control system saves a copy of code that you've deleted, so you can always get it back later if you need it. Commented-out code is confusing to someone reading the code later.